local/tractus-x-edc/config/customer/control-plane.properties

```diff
@@ -4,90 +4,89 @@ web.http.default.path=/api
-edc.ids.id=urn:connector:customer-control-plane
-edc.ids.security.profile=base
-edc.ids.maintainer=http://customer-control-plane
-edc.ids.curator=http://customer-control-plane
-edc.ids.catalog.id=urn:catalog:default
 
-# /api/v1/ids is the default HTTP IDS path
-ids.webhook.address=http://customer-control-plane:8184
+edc.participant.id=BPNL4444444444XX
+edc.api.auth.key= ${EDC_API_PW}
+edc.dsp.callback.address=http://customer-control-plane:8184/api/v1/dsp
+
 
 edc.hostname=customer-control-plane
 
-# set via .env
-#edc.api.auth.key=password
-
-# OAuth / DAPS related configuration
-edc.ids.endpoint=http://customer-control-plane:8184/api/v1/ids
-## this may relate to version > 0.1.3
-edc.oauth.endpoint.audience=http://customer-control-plane:8184/api/v1/ids/data
-## starting from 0.4.0 with new protocol
-#edc.oauth.endpoint.audience=http://consumer-control-plane:8184/api/v1/dsp
-edc.oauth.token.url=http://omejdn-daps:4567/token
-edc.oauth.client.id=customer
-edc.oauth.provider.jwks.url=http://omejdn-daps:4567/jwks.json
-edc.oauth.provider.audience=idsc:IDS_CONNECTORS_ALL
-edc.oauth.public.key.alias=customer-cert
-edc.oauth.private.key.alias=customer-key
-edc.ids.validation.referringconnector=false
+tx.ssi.oauth.token.url=http://keycloak:8080/realms/miw_test/protocol/openid-connect/token
+tx.ssi.oauth.client.id=${CUSTOMER_OAUTH_CLIENT_ID}
+tx.ssi.oauth.client.secret.alias=${CUSTOMER_OAUTH_SECRET_ALIAS}
+tx.ssi.miw.url=http://miw
+tx.ssi.miw.authority.id=BPNL000000000000
+tx.ssi.endpoint.audience=http://customer-control-plane:8184/api/v1/dsp
 
 # HashiCorp vault related configuration
 edc.vault.hashicorp.url=http://vault:8200
-# set via docker-compose .env
-#edc.vault.hashicorp.token=
 edc.vault.hashicorp.health.check.enabled=false
+edc.vault.hashicorp.token= ${VAULT_DEV_ROOT_TOKEN_ID}
 
 # Data-Plane configuration
 edc.transfer.proxy.endpoint=http://customer-data-plane:8285/api/public/
-edc.transfer.proxy.token.signer.privatekey.alias=customer-key
-edc.transfer.proxy.token.verifier.publickey.alias=customer-cert
+edc.transfer.proxy.token.signer.privatekey.alias=${CUSTOMER_PRIVATE_KEY_ALIAS}
+edc.transfer.proxy.token.verifier.publickey.alias=${CUSTOMER_PUBLIC_KEY_ALIAS}
 
 # Data-Plane selector configuration
-edc.dataplane.selector.edchttp.url=http://customer-data-plane:8299/api/dataplane/control
+edc.dataplane.selector.edchttp.url=http://customer-data-plane:8299/control/transfer
 edc.dataplane.selector.edchttp.sourcetypes=HttpData
-edc.dataplane.selector.edchttp.destinationtypes=HttpProxy
+edc.dataplane.selector.edchttp.destinationtypes=HttpData,HttpProxy
 edc.dataplane.selector.edchttp.properties={"publicApiUrl" : "http://customer-data-plane:8285/api/public/"}
 
-# backend receiver for static Endpoint Data References
-edc.receiver.http.endpoint=http://backend-app:8080
+#edc.receiver.http.endpoint=http://customer-backend:8081/catena/edrendpoint
+edc.receiver.http.endpoint=http://host.docker.internal:4000
```

Comment:
Is that needed to communicate with the other network? Wouldn't it be better to reference the network?

Reply:
Oh sorry, that was a leftover from previous testing. It`s actually not effectively doing anything at the moment, since we're now specifying the edr endpoint within the transfer request body. I will remove that line 49.